client-templates/artifactory/accept.json.sample

```diff
@@ -2,7 +2,8 @@
   "public": [
     {
       "method": "any",
-      "path": "/*"
+      "path": "/*",
+      "stream": true
```

Comment:
Streaming is required for artifactory, as large chunks of binary data will be passed through